tasks/main.yml

```diff
@@ -7,15 +7,20 @@
   sudo: yes
   apt: pkg=apache2 state=latest
 
-- name: Install Apache Modules
-  apache2_module: state=present name={{ item }}
-  notify: restart apache
-  with_items:
-    - rewrite
-    - vhost_alias
-    - headers
-    - expires
-    - filter
+- name: Install Apache Modules rewrite
```

Comment:
Set a comment of why this is this way. If not, someone will try to use with_items at some point.